templates/ts-apollo-mongodb-backend/README.md

```diff
@@ -1,35 +1,13 @@
-## Graphback Apollo+MongoDB Runtime Template
+## Graphback MongoDB + Apollo Client Runtime Template
 
-Template showcases Graphback Runtime capabilities using
-resolver layer created in-memory as opposed to codegeneration.
+A template which showcases Graphback Runtime capabilities using a resolver layer created in-memory as opposed to codegeneration.
```

Comment:
Graphback no longer generates code resolvers, i.e. it is a runtime-only server solution. The documentation should no longer mention resolver code generation.

Reply:
What I mean to say is that Graphback does not generate resolver **files** - it does generate resolvers which are completely in memory and are output as `resolvers` in [`buildGraphbackAPI`](https://graphback.dev/docs/api/graphback/interfaces/_buildgraphbackapi_.graphbackapi#resolvers).